charts/cinder-csi-plugin/templates/controllerplugin-statefulset.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   labels:
     {{- include "cinder-csi.controllerplugin.labels" . | nindent 4 }}
 spec:
+  serviceName: csi-cinder-controller
```

Comment:
Ideally it should be name of headless service , could you add the service as well

Reply:
Headless service added